### ISSUE TITLE
Does this library support regression?

### ISSUE THREAD

**Issue by johnkarasev** [davechallis/rust-xgboost#3]
Xgboost provides regression, does this library include that feature?

**Comment by davechallis** [comment]
Yup, you'd generally just need to select one of the [regression objectives](https://docs.rs/xgboost/0.1.4/xgboost/parameters/learning/enum.Objective.html) when configuring the learning parameters (e.g. [`RegLinear`](https://docs.rs/xgboost/0.1.4/xgboost/parameters/learning/enum.Objective.html#variant.RegLinear)).
